code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -61,7 +63,34 @@
 			name = "improvised explosive"
 			active = 0
 			det_time = rand(30,80)
+	else
+		if(assembled == 2 && current_shrapnel < max_shrapnel )
+			if(I.is_shrapnel && I.w_class == W_CLASS_TINY || I.w_class == W_CLASS_TINY || I.is_shrapnel)
+				if(user.drop_item(I, src))
+					shrapnel_list.Add(I)
+					I.forceMove(src)
```

Comment:
This line is not necessary because the second argument of `drop_item(I, loc)` already puts it inside the can.